src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -80,7 +80,33 @@ public Boolean apply(WebDriver from) {
       @Override
       public String toString() {
         return String.format("value ('%s') to be present in element found by %s", value,
-            element.getTagName());
+                             element.getTagName());
+      }
+    };
+  }
+
+  /**
+   * An expectation for checking if the given text is present in the specified element.
+   *
+   * @author Michal Nowierski
```

Comment:
I'm not sure do we have a guideline on whether we're using author info? Seems its not set in other places in this file. In general I'm not a big fan of the @author annotation as after some time when the code is changed by multiple people it's never accurate.